great_expectations/checkpoint/checkpoint.py

```diff
@@ -1073,7 +1082,7 @@ def _run_default_validation_operator(
             )
 
         for batch in assets_to_validate:
-            if not isinstance(batch, (tuple, DataAsset, Validator)):
+            if not isinstance(batch, (tuple, DataAsset, _get_validator_class())):
```

Comment:
Don't think this is the right way to go about it - can't we do an inline import?

Reply:
The idea here is to avoid circular dependency, so `Validator.__class__.__name__` would not work (we are trying to avoid depending on the actual `Validator`, except for `TYPE_CHECKING`.  Thanks.